azure/v13.1.0/README.md

```diff
@@ -0,0 +1,78 @@
+# :zap: Giant Swarm Release v13.1.0 for Azure :zap:
+
+This is next minor tenant cluster release for Node Pools Clusters.
+
+Release brings two new features that can be enabled by customers:
+
+- Auto Terminate Unhealth Nodes feature enables customers to discard unhealthy kubernetes nodes in favour of new ones added to the cluster. Follow [the guide](https://docs.giantswarm.io/basics/automatic-termination-of-bad-nodes/) to learn how to enable and disable this feature via annotations on Cluster CR.
+
+- [Node Pools Auto Scaling](https://docs.giantswarm.io/basics/nodepools/#autoscaling) feature allows customers to set up minimum and maximum number of nodes per node pool used by autoscaler app installed by default.
+
+
+## Change details
+
+
+### azure-operator [5.1.0](https://github.com/giantswarm/aws-operator/releases/tag/v5.1.0)
+
+#### Changed
+- Only submit Subnet ARM deployment when Subnet name or Subnet CIDR change.
+- Use controller-runtime instead of typed clients.
```

Comment:
could you add 'why' was this changed?

Reply:
uhm, they are kind of "internal optimizations" that have little to no impact for the customer. Should we remove them from here? They would be available in the component's changelog if the customer is interested.

Reply:
Let's talk on Monday 😁

Reply:
Aaand we didn't talk about this :D